### ISSUE TITLE
Port Drupal updates

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/date_repeat#14]
It would be worth scanning through the commits to the Drupal version since the fork to port those fixes, too.

**Comment by yorkshire-pudding** [comment]
@laryn Are we talking about from 2015 when the earliest issue is?  I can't work out when the module was forked.
[Doing a search on the date project for date_repeat](https://git.drupalcode.org/project/date/-/commits/7.x-2.x?utf8=%E2%9C%93&search=date_repeat) shows just two commits since them.

**Comment by laryn** [comment]
@yorkshire-pudding Yes, it looks like the last commit from D7 was Sep. 7, 2015 (at least a glance):

 - https://github.com/backdrop-contrib/date_repeat/commits/1.x-2.x

**Comment by laryn** [comment]
These commit histories show a few more relevant commits from D7 since then:

 - https://git.drupalcode.org/project/date/-/commits/7.x-2.x/date_repeat
 - https://git.drupalcode.org/project/date/-/commits/7.x-2.x/date_repeat_field
